lib/core/src/Cardano/Wallet/Transaction.hs

```diff
@@ -76,6 +77,21 @@ data TransactionLayer t k = TransactionLayer
         -- This expects as a first argument a mean to compute or lookup private
         -- key corresponding to a particular address.
 
+    , mkDelegationCertTx
```

Comment:
I thought `mkCertificateTx` was unnecessarily vague name given that there exist multiple types of certificates. So I thought something with `delegation` or `stake` in it would be nice.